app/scripts/views/mixins/password-mixin.js

```diff
@@ -53,6 +54,37 @@ define(function (require, exports, module) {
         // IE8 blows up when changing the type of the input field. Other
         // browsers might too. Ignore the error.
       }
+    },
+
+    onPasswordKeyUp: function (event) {
+      var values = [];
+      values.push(this.getElementValue('.password').length);
+
+      // Check to see if change password fields are visible.
+      // If any field does not have 8 chars, display warning.
+      if (this.getElementValue('#new_password') || this.getElementValue('#new_password') === '') {
+        values.push(this.getElementValue('#new_password').length);
+      }
+
+      if (this.getElementValue('#old_password') || this.getElementValue('#old_password') === '') {
+        values.push(this.getElementValue('#old_password').length);
+      }
+
+      var val = Math.min.apply(Math, values);
+
+      if (val < 8) {
```

Comment:
Even if it's hard coded in the strings, we do have a [Constants.PASSWORD_MIN_LENGTH](https://github.com/mozilla/fxa-content-server/blob/468811c5b93790b8691d774e851b7814047cd6eb/app/scripts/lib/constants.js#L72) to use in code.